src/main/java/com/urlshortner/discovery/autoconfig/SpringCloudDiscoveryAutoConfig.java

```diff
@@ -0,0 +1,18 @@
+package com.urlshortner.discovery.autoconfig;
```

Comment:
can we mention this configuration directly in `application.yaml`?

Reply:
Not sure....🤷‍♀️